host/CMakeLists.txt

```diff
@@ -174,6 +174,8 @@ if (UNIX)
   target_compile_options(oehost PRIVATE
     -Wno-attributes -Wmissing-prototypes -fPIC ${PLATFORM_FLAGS})
   target_compile_definitions(oehost PRIVATE _GNU_SOURCE)
+  target_compile_options(oehost PUBLIC -fstack-protector-strong)
```

Comment:
Just a style thing, but I tend to prefer combining these, more like:

```
target_compile_options(oehost
  PRIVATE -Wno-attributes -Wmissing-prototypes -fPIC ${PLATFORM_FLAGS}
  PUBLIC -fstack-protector-strong)
```

Just to keep the number of calls to `target_compile_options` low so they're easier to find.

Reply:
Updated.